assets/app/view/create_game.rb

```diff
@@ -342,7 +358,7 @@ def mode_selector
 
     def mode_input(mode, text)
       click_handler = lambda do
-        store(:mode, mode, skip: true)
+        store(:mode, mode) # , skip: true)
```

Comment:
why was this changed? should you remove the comment

Reply:
To hide the minimum players input when hotseat is selected. I will verify it is still needed and if so remove the commented out portion.

Reply:
Not needed. Reverted.